internal/pkg/workspace/workspace.go

```diff
@@ -160,6 +162,66 @@ func (ws *Workspace) ListWorkloads() ([]string, error) {
 	})
 }
 
+// Pipeline holds identifying information about a pipeline.
+type Pipeline struct {
```

Comment:
What do you think of renaming it to:
```suggestion
// PipelineManifest holds identifying information about a pipeline manifest file.
type PipelineManifest struct {
     Name string // Name of the pipeline inside the manifest file.
     Path string // Absolute path to the manifest file for the pipeline.
```
so that it reads `workspace.PipelineManifest` instead of `workspace.Pipeline`

Reply:
Okay! 
Then I'll also rename `manifest.PipelineManifest` to `manifest.Pipeline` now rather than later, to prevent confusion.